hat/app/org/hatdex/hat/api/controllers/Authentication.scala

```diff
@@ -223,4 +225,100 @@ class Authentication @Inject() (
         Future.successful(Unauthorized(Json.toJson(ErrorMessage("Invalid Token", "Token does not exist"))))
     }
   }
+
+  /*
+    BEGIN: hat-claim
+    Entire Section below is on HAT Claim.
+    TODO: Refactor to New File?
+   */
+  /**
+   * Sends an email to the owner with a link to claim the hat
+   */
+  def claim(applicationId: String): Action[AnyContent] = SecuredAction(WithRole(Owner())).async { implicit request =>
+    val email = request.dynamicEnvironment.ownerEmail
+    val response = Ok(Json.toJson(SuccessResponse("You will shortly receive an email with claim instructions")))
+
+    applicationsService.applicationStatus().flatMap { applications =>
+      applications.find(_.application.id.equals(applicationId)) match {
+        case Some(app) =>
+          usersService.listUsers.map(_.find(_.roles.contains(Owner()))).map {
```

Comment:
What's the point to search for owner user if the result is not used afterwards?

Reply:
Fixed